tests/controllers/test_cir_controller.py

```diff
@@ -88,79 +88,3 @@ def test_ApiError_thrown_when_connection_error(self, mock_get):
-
-    @patch("requests.Session.get")
-    def test_ApiError_thrown_when_timeout(self, mock_get):
-        with self.app.app_context():
-            current_app.config["CIR_API_URL"] = TEST_CIR_URL
-
-            mock_get.side_effect = requests.exceptions.Timeout("Timeout")
-
-            with self.assertRaises(ExternalApiError) as context:
-                get_cir_service_status()
-            self.assertEqual(context.exception.error_code, ErrorCode.API_TIMEOUT_ERROR)
-
-    @responses.activate
-    def test_ApiError_thrown_when_not_200(self):
-        with self.app.app_context():
-            current_app.config["CIR_API_URL"] = TEST_CIR_URL
-
-            responses.add(
-                responses.GET,
-                f"{TEST_CIR_URL}/status",
-                status=401,
-            )
-
-            with self.assertRaises(ExternalApiError) as context:
-                get_cir_service_status()
-            self.assertEqual(context.exception.error_code, ErrorCode.API_UNEXPECTED_STATUS_CODE)
-
-    @responses.activate
-    def test_ApiError_thrown_when_404(self):
-        with self.app.app_context():
-            current_app.config["CIR_API_URL"] = TEST_CIR_URL
-            current_app.config["CIR_API_API_PREFIX"] = TEST_CIR_API_PREFIX
-
-            responses.add(
-                responses.GET,
-                f"{TEST_CIR_URL}{TEST_CIR_API_PREFIX}{cir_url_query_parameters}",
-                status=404,
-            )
-
-            with self.assertRaises(ExternalApiError) as context:
-                get_cir_metadata(survey_ref, formtype)
-            self.assertEqual(context.exception.error_code, ErrorCode.NOT_FOUND)
-
-    @responses.activate
-    def test_ApiError_thrown_when_content_not_json(self):
-        with self.app.app_context():
-            current_app.config["CIR_API_URL"] = TEST_CIR_URL
-
-            responses.add(
-                responses.GET,
-                f"{TEST_CIR_URL}/status",
-                status=200,
-                body="not_json_despite_content_type_saying_it_is",
-                content_type="application/json",
-            )
-
-            with self.assertRaises(ExternalApiError) as context:
-                get_cir_service_status()
-            self.assertEqual(context.exception.error_code, ErrorCode.API_UNEXPECTED_CONTENT)
-
-    @responses.activate
-    def test_ApiError_thrown_when_content_type_not_application_json(self):
-        with self.app.app_context():
-            current_app.config["CIR_API_URL"] = TEST_CIR_URL
-
-            responses.add(
-                responses.GET,
-                f"{TEST_CIR_URL}/status",
-                status=200,
-                body="this_is_plain_text",
-                content_type="text/plain",
-            )
-
-            with self.assertRaises(ExternalApiError) as context:
-                get_cir_service_status()
-            self.assertEqual(context.exception.error_code, ErrorCode.API_UNEXPECTED_CONTENT_TYPE)
```

Comment:
These should still be tested. Maybe moved to a connection helper tester suite.

Reply:
Dope, either I can write test super quick or i didn't push them, I will take either